packages/@guidesmiths/cuckoojs-schematics/README.md

```diff
@@ -26,3 +26,48 @@ npm publish
 ```
 
 That's it!
+
+## Available schematics
```

Comment:
Really nice docs!! Thanks!

Reply:
It's just a proposal. I'm not sure if we should keep a readme on each component or having a big one 🤔

Reply:
As they are two separate npm packages, we need to keep different READMEs

Reply:
I mean a readme per schematics, but maybe it will be too many readmes